eng/pipelines/templates/steps/npm-release-task.yml

```
@@ -16,7 +16,10 @@ steps:
           Intent: 'PackageDistribution'
           ContentType: 'npm'
           FolderLocation: ${{parameters.PathToArtifacts}}
-          Owners: $(Build.RequestedForEmail)
+          ${{ if ne(variables['Build.RequestedForEmail'], '') }}:
+            Owners: $(Build.RequestedForEmail)
+          ${{ else }}:
+            Owners: 'azuresdk@microsoft.com'
```

```suggestion
          Owners: coalesce(variables['Build.RequestedForEmail'], 'azuresdk@microsoft.com')
```

Please test but I believe something like that should work.